src/chrome-helper.ts

```diff
@@ -759,10 +759,6 @@ export const getProtocolJSON = async () => {
 
 export const killAll = async () => {
   await Promise.all(runningBrowsers.map((browser) => closeBrowser(browser)));
-
-  runningBrowsers = [];
```

Comment:
This because `closeBrowser` takes care of removing them? Just wanted to double check